### DISCUSSION TITLE
XLS / XLSX Convertion - Whole Sheet Export Option

### DISCUSSION THREAD

**Discussion by mjbergman92** [gotenberg/gotenberg#1027] [Q&A] [answered]
In LibreOffice, there is a "Whole sheet export" option. Is this possible using the gotenberg API?

The conversion for the xls and xlsx files is terrible without this option. It shifts the columns and such all around, creates nearly blank pages, etc.

Example from the LibreOffice GUI

![image](https://github.com/user-attachments/assets/d28afc09-644b-46b4-ba81-57d83625c993)

**Comment by gulien** [comment, marked as answer]
Hello @mjbergman92  👋

`singlePageSheets`?

**Reply by mjbergman92** [reply]
is there a way to set these values as environment variables? or can they only be set for individual API requests?

**Reply by mjbergman92** [reply]
The reason I was asking is because I am using paperless-ngx and it is not an option from their configuration; however, I did figure out a proxy for the requests, so my issue is solved!

Yes, singlePageSheets was exactly what "Whole Sheet Export" option in the GUI of LibreOffice export is.

**Reply by gulien** [reply]
No currently there is no way to set default values via environment variables! That being said, I think it should be an option in paperless-ngx directly.

Anyway, glad it worked out for you!
